saltstack/freeipa/salt/ipaconsistency/init.sls

```diff
@@ -12,7 +12,7 @@
     - name: /usr/local/lib/python3.8/site-packages/checkipaconsistency/main.py
     - source: salt://{{ slspath }}/scripts/main.py
     - mode: 644
-    - onlyif: ls -la /usr/local/lib/python3.8/site-packages/
+    - onlyif: ls -la /usr/local/lib/python3.8/site-packages/ && ! ls -la /usr/local/lib/python3.6/site-packages/
```

Comment:
Why is this necessary? How did it work before? Was this actually ever executed as I see FreeIPA used to have Python 3.6 only?

Reply:
When there is both Python 3.6 and Python 3.8, the 3.8 salt state failed, because the site-packages directory was only created for 3.6. I added this as a quick workaround to solve this. I am not sure if we need the `checkipaconsistency/main.py` file in both site-packages directories.